pyproject.toml

```diff
@@ -59,10 +59,26 @@ exclude_lines = [
 ]
 
 [tool.pyright]
-include = ["examples", "hikari"]
```

Comment:
This removes examples, which we dont want

Reply:
Rillrate is uncheckable so I'll exclude only that and keep all other examples.